docs/changelog.rst

```diff
@@ -7,7 +7,11 @@ Changelog
 This document lists changes between released versions of
 pwned-passwords-django.
 
-2.0 -- released 2023-??-??
+Unreleased
+----------
+Exports type hints for consumption by ``mypy`` and other typecheckers.
+
+2.0 -- released 2023-03-26
```

Comment:
Separate PR for this one-liner if you're so inclined
https://github.com/ubernostrum/pwned-passwords-django/pull/39